lib/new_relic/agent/log_event_aggregator.rb

```diff
@@ -63,8 +64,9 @@ def record(formatted_message, severity)
           end
         end
 
+        return if severity_too_low?(severity)
```

Comment:
My mind really wanted the inverse with something like `return unless severity_level_permitted?` but after reading through the whole class it caught on.